app/services/reports/facility_progress_service.rb

```diff
@@ -81,15 +81,15 @@ def dimension_combinations_for(indicator)
 
     attr_reader :diabetes_enabled
 
-    memoize def daily_registrations_grouped_by_day
+    def daily_registrations_grouped_by_day
```

Comment:
Why remove `memoize`? Additionally, can we remove the `include  Memery` at the top now?

Reply:
I've added the reason to remove memoize in the [description](https://github.com/simpledotorg/simple-server/pull/3594#issue-1161228309)

tl;dr: Using memoize can cause confusion about when the actual numbers got updated.